monitoring_101.rst

```diff
@@ -42,10 +42,18 @@ Since Nagios is so configurable, it can often be difficult to configure for the
 It can use many files for configuration, and a single syntax error will prevent the system from starting.
 Additionally, the open-source version does not natively support adding and removing hosts dynamically; the configuration needs to be modified, and the server reloaded to add or remove a host.
 
+Icinga/Icing2
+------
```

Comment:
This line is the one too short, does not appear to be changed in the second commit.